### DISCUSSION TITLE
Auto add data to user_meta_data_raw when Profile was updated

### DISCUSSION THREAD

**Discussion by IRediTOTO** [supabase/supabase#2823] [Questions] [answered]
Hello, I read this post `https://supabase.io/docs/guides/auth`
But I want to improve more
I want when I run query update for one colum  row for profile one or more column in there will auto add to metadata, just some data, exclude secret data. 
Thank you for any help.

**Comment by steve-chavez** [comment, marked as answer]
Hey @IRediTOTO,

> But I want to improve more
I want when I run query update for one colum row for profile one or more column in there will auto add to metadata, just some data, exclude secret data.

I think this would be more fit for a feature request on [GoTrue](https://github.com/supabase/gotrue/issues), that's the main component behind auth. I suggest you open an issue there.
